lua/ulx/modules/sh/cfc_trainfuck.lua

```diff
@@ -0,0 +1,31 @@
+CATEGORY_NAME = "Fun"
+
+local function trainFuck( callingPlayer, targetPlayers )
+	local Y = 1
+	for I,ply in pairs(targetPlayers) do
+		local train = ents.Create("prop_physics")
+		train:SetModel("models/props_trainstation/train001.mdl")
+		train:SetPos(ply:GetPos()+Vector(0,-1000,300))
+		train:Spawn()
+		
+		local phys = train:GetPhysicsObject()
+		
+		if (IsValid(phys)) then
+			if ply:GetMoveType(MOVETYPE_NOCLIP) then
+				ply:SetMoveType(MOVETYPE_WALK)
+			end
+			ply:ExitVehicle()
+			phys:ApplyForceCenter((ply:GetPos()-train:GetPos())*1000000000)
+			timer.Create("trainFuckTimer"..Y,1.5,1,function()
```

Comment:
you could use timer.Simple here.
https://wiki.facepunch.com/gmod/timer.Simple

Reply:
timer.Simple doesn't allow to set a name on it, this will cause it to only run it on the last train created.

Reply:
That's precisely why it's a good choice here, it just runs the one time and then disappears, which is what you're looking for here. We don't need to keep track of it so we just make an anonymous timer and then we're good.

Reply:
I messed up, you're right. I'll fix it right away.